### PR TITLE
Migration to config MFD

### PR DESCRIPTION
**Pull request by mattysmith22**
* Old MPD / UFD systems implemented

# UFD

* Removed old UFD
* Added config-based UFD
* Added ACRE support to UFD

# MPD

* Removed old MPD geometry
* Implemented following MPD pages
  * MENU
  * WPN
  * FUEL
  * FLT
  * WCA
  * TSD
  * FCR
  * ENG
  * ACE
  * DMS
  * DTU
* MPDs work on both screens
* Implemented A/C Button
* Implemented KU interaction

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_ku/config.cpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CfgPatches
+{
+	class fza_ah64_ku
+	{
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_ku/config.cpp:4]: needs updating to match control author list
required version needs updating
required addon needs updating

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/config.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class CfgPatches
+{
+	class fza_ah64_mpd
+	{
+        units[] = {};
+		author = "Franze, Nodunit, Sacha 'Voodooflies' Oropeza, Keplager & Community";
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/config.cpp:6]: needs updating to match control author list
required version needs updating
required addon needs updating

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/fn_drawMap.sqf [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /* ----------------------------------------------------------------------------
-Function: fza_fnc_mpdUpdateMap
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/fn_drawMap.sqf:2]: is thsi meant to be deleted

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/fn_handleZoom.sqf [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /* ----------------------------------------------------------------------------
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/fn_handleZoom.sqf:1]: Tsd range settings need hooking up

File: @AH-64D Apache Longbow/Addons/fza_ah64_ufd/config.cpp [hunk 5]
```diff
@@ -0,0 +1,14 @@
+class CfgPatches
+{
+	class fza_ah64_ufd
+	{
+        units[] = {};
+		author = "Franze, Nodunit, Sacha 'Voodooflies' Oropeza, Keplager & Community";
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_ufd/config.cpp:6]: needs updating to match control author list
required version needs updating
required addon needs updating

**Review by Ollieollieolllie — CHANGES_REQUESTED** [review]

**Comment by mattysmith22** [comment]
Review changes complete, merged to update

**Review by Ollieollieolllie — APPROVED** [review]
Great work

**Pull request merged**